pytest.ini

```diff
@@ -34,3 +34,6 @@ filterwarnings=
 
 	# https://github.com/pypa/setuptools/issues/2497
 	ignore:.* is an invalid version and will not be supported::pkg_resources
+
+	# This will result in an error in the integration tests if not ignored:
+	ignore:.*Coverage disabled via --no-cov switch!.*
```

Comment:
`pytest-cov` emits a warning if it is disabled. That warning is converted to error by the strict `pytest` configuration.

Reply:
Instead, you can pass `-p no:cov` to the pytest run.